demo_path_routing/index.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
```

Comment:
I think the demo ui is minimal and straightforward as it should be! Could possibly benefit from a please wait page, maybe even more so than before now that there is a slot for custom info. 

`Please wait while <container_name> starts...` ?

The demo tools start up fairly quickly though (after their images are cached locally), so maybe it would be overkill?

Reply:
If you think this is an important part of the functionality which we should demo, sure, but I don't think we need to make it prettier for its own sake. I could be thinking about it wrong.